Formula/kibana.rb

```diff
@@ -15,8 +15,9 @@ class Kibana < Formula
   end
 
   resource "node" do
-    url "https://nodejs.org/dist/v6.11.1/node-v6.11.1.tar.xz"
```

Comment:
why not https://nodejs.org/dist/v6.11.5/node-v6.11.5.tar.xz ?

Reply:
Because the source tarballs do not support `make lint` and the build now fails if linting is not available; see nodejs/node#13658.

Reply:
ok